app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
+    @post = Post.find_by_id(params[:post_id])
+    @comments = @post.comments
+    render json: @comments
+  end
+
+  def create
+    @post = Post.find_by_id(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author = current_user
+
+    if @comment.save
+      render json: @comment
+    else
+      render json: { errors: @comment.errors.full_messages }
+    end
+  end
```

Comment:
- [ ] If I hit this API `/api/v1/users/2/posts/1/comments` then give me the following error :point_down: 

![Screenshot from 2023-08-05 02-02-35](https://github.com/anbehindY/Dev-Blog/assets/45226848/6ff42ca7-3f77-4ceb-8bbb-6bf6226cd939)

- [ ] So kindly fix this error as well :slightly_smiling_face: